admin/class-taxonomy-presenter.php

```diff
@@ -0,0 +1,317 @@
+				'description' => sprintf( esc_html__( 'The Breadcrumbs title is used in the breadcrumbs where this %s appears.', 'wordpress-seo' ), $this->term->taxonomy ),
+				'type'        => 'text',
+				'options'     => '',
+				'hide'        => $this->options['breadcrumbs-enable'] !== true,
+			),
+			'noindex'  => array(
+				'label'       => sprintf( __( 'Noindex this %s', 'wordpress-seo' ), $this->term->taxonomy ),
+				'description' => sprintf( esc_html__( 'This %s follows the indexation rules set under Metas and Titles, you can override it here.', 'wordpress-seo' ), $this->term->taxonomy ),
+				'type'        => 'select',
+				'options'     => $noindex_options,
+			),
+			'sitemap_include'  => array(
+				'label'       => sprintf( __( 'Include in sitemap?', 'wordpress-seo' ), $this->term->taxonomy ),
+				'description' => '',
+				'type'        => 'select',
+				'options'     => $this->sitemap_include_options,
+			),
+		);
+
+		unset( $current, $no_index_options );
+
+		return $fields;
+	}
+
+	/**
+	 * Returning the fields for the social media tab
+	 *
+	 * @return array
+	 */
+	public function social_fields() {
+		$social_networks = array(
+			'opengraph'  => array(
+				'network' => 'opengraph',
+				'label'   => __( 'Facebook', 'wordpress-seo' ),
+				'size'    => '1200 x 628',
+			),
+			'twitter'    => array(
+				'network' => 'twitter',
+				'label'   => __( 'Twitter', 'wordpress-seo' ),
+				'size'    => '1024 x 512',
+			),
+			'googleplus' => array(
+				'network' => 'google-plus',
+				'label'   => __( 'Google+', 'wordpress-seo' ),
+				'size'    => '800 x 1200',
+			),
+		);
+
+		$fields = array();
+		foreach ( $social_networks as $option => $settings ) {
+			if ( true === $this->options[ $option ] ) {
+				$fields_to_push = array(
+					$settings['network'] . '-title' => array(
+						'label' 	  => sprintf( __( '%s Title', 'wordpress-seo' ), $settings['label'] ),
+						'description' => sprintf( esc_html__( 'If you don\'t want to use the title for sharing on %1$s  but instead want another title there, write it here.', 'wordpress-seo' ), $settings['label'] ),
+						'type'        => 'text',
+						'options'     => array( 'class' => 'large-text' ),
+					),
+					$settings['network'] . '-description' => array(
+						'label'       => sprintf( __( '%s Description', 'wordpress-seo' ), $settings['label'] ),
+						'description' => sprintf( esc_html__( 'If you don\'t want to use the meta description for sharing on %1$s but want another description there, write it here.', 'wordpress-seo' ), $settings['label'] ),
+						'type'        => 'textarea',
+						'options'     => '',
+
+					),
+					$settings['network'] . '-image' => array(
+						'label'       => sprintf( __( '%s Image', 'wordpress-seo' ), $settings['label'] ),
+						'description' => sprintf( esc_html__( 'If you want to use an image for sharing on %1$s, you can upload / choose an image or add the image URL here.', 'wordpress-seo' ), $settings['label'] ) . '<br />' .
```

Comment:
Please add translator comments to explain what %1$s is.
Why did you use %1$s instead of %s here?